bundles/org.openhab.core/src/main/java/org/openhab/core/util/ColorUtil.java

```diff
@@ -296,9 +314,19 @@ public static HSBType xyToHsbFine(double[] xy, Gamut gamut) throws IllegalArgume
     }
 
     /**
-     * Inner method to support both rounded and fine versions of xyToHsb().
+     * Transform <a href="https://en.wikipedia.org/wiki/CIE_1931_color_space">CIE 1931</a> `xy` format to
+     * <a href="https://en.wikipedia.org/wiki/RGB_color_model">HSV</a> as an RGB array of double values.
+     *
+     * See <a href=
+     * "https://developers.meethue.com/develop/application-design-guidance/color-conversion-formulas-rgb-to-xy-and-back/">Hue
+     * developer portal</a>.
+     *
+     * @param xy the CIE 1931 xy color, x,y[,Y] between 0.0000 and 1.0000. <code>Y</code> value is optional.
+     * @param gamut the gamut supported by the light.
+     * @return an array of doubles containing the RGB elements.
+     * @throws IllegalArgumentException when input array has wrong size or exceeds allowed value range.
      */
-    private static double[] xyToRgb(double[] xy, Gamut gamut) throws IllegalArgumentException {
+    public static double[] xyToRgb(double[] xy, Gamut gamut) throws IllegalArgumentException {
```

Comment:
Please don't introduce a third way of representing RGB. We have `int[]` (for rounded 0-255) and `PercentType[]` for 0-100 with `BigDecimal` precision. I don't see any benefit in cluttering API by adding a `double[]`.

Reply:
If you allow the 'fine' method to exist, then this method can be private. But if you want 'fine' to be removed, then making this public allows users to do the 'fine' themselves externally.

Reply:
Why? I don't see a use-case for `xyToRgb` on the openHAB context, but if you absolutely want such a method, it should return `PercentType[]` like the `hsbToRgbPercent`, which also returns the "exact" values.

Reply:
Ok.